site/docs/patterns/button-bar.mdx

```diff
@@ -150,14 +140,12 @@ The container and the type of scenario determine the position of actions (CTA, p
   <Diagram
     src="/img/patterns/button-bar/vert-nav-layout-4.png"
     alt="Example showing a dialog with a form and button bar underneath. The core actions are on the right."
-    caption="Example showing a dialog with a form and button bar underneath. The core actions are on the right."
-    contentPosition="bottom"
     border
   />
 
 <ExampleContainer type="positive" customPillText="Best practices">
   - Group core onward actions for the form to ensure they remain accessible on browser zoom (e.g., **Previous**, **Next**, **Cancel** or **Submit**).
-  - Display secondary actions that aren’t core to the workflow at the opposite end of the button bar.
+  - You should display transparent actions that aren’t core to the workflow at the opposite end of the button bar.
```

Comment:
This doesn't feel equivalent. "secondary actions" are an actual thing "transparent actions" aren't.  "core actions" are used to describe the other type of actions.

Reply:
Also secondary action is already used previously.